packages/mobx-binder-dayjs/src/conversion/DayjsConverter.ts

```diff
@@ -18,8 +18,13 @@ export class DayjsConverter implements Converter<BinderValidationResult, string
         return typeof formats === 'string' ? formats : formats[0]
     }
 
+    private isNotEmpty(value: string) {
+        const valueWithoutSpace = value.replace(/ /g, '');
+        return valueWithoutSpace.length > 2
+    }
+
     public convertToModel(value?: string): Dayjs | undefined {
-        if (!!value) {
+        if (!!value && this.isNotEmpty(value)) {
```

Comment:
We could also just add a TrimFilter in front of the converter, so there is no need for special handling of such values here